=1.3.17

```diff
@@ -0,0 +1,11 @@
+Looking in indexes: https://pypi.tuna.tsinghua.edu.cn/simple
```

Comment:
This file seems irrelevant.

Reply:
Yes... This is weired. I tried the coverage test on my device, and it reported "need MMCV>=3.1.17".
It may appear when I reinstall MMCV. I'll delete it